### PR TITLE
Allow photon PDF to reports

### PR DESCRIPTION
**Pull request by niclaurenti**
Allow photon PDF in reports through the argument `--photon` in `vp-comparefits`

### CODE REVIEW

**Comment by niclaurenti** [comment]
This is the output of the report:
https://vp.nnpdf.science/7DxRLas3S_a3zqixFEzADg==

File: validphys2/src/validphys/scripts/vp_comparefits.py [hunk 1]
```diff
@@ -76,6 +76,11 @@ def add_positional_arguments(self, parser):
             '--lite',
             help="Smaller version of the usual comparefit fit",
             action='store_true')
+        parser.add_argument(
+            '-p',
+            '--photon',
+            help="Add photon PDF to the report",
```
Comment (scarlehoff) [inline validphys2/src/validphys/scripts/vp_comparefits.py:82]:
```suggestion
            help="Use LUX basis (which include the photon) for the report",
```

to be a bit more specific on what the flag is doing.

**Review by scarlehoff — APPROVED** [review]
I'm fine with this soluton

**Comment by RoyStegeman** [comment]
I think you can just add the photon to `FLAVOUR` without changing `default_elements` and do `copy.deepcopy` for the `LUX_FLAVOUR` where the photon is in the `default_elements`. Same as we did for evolution

**Review by RoyStegeman — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**